Podfile

```diff
@@ -9,7 +9,7 @@ target 'ooniprobe' do
     pod 'MBProgressHUD'
     pod 'DZNEmptyDataSet'
     pod 'lottie-ios'
-    pod 'SharkORM'
+#    pod 'SharkORM'
```

Comment:
I believe we have the option to point the `pod` to the github repository rather than to a pod, which is what we do with `mkall-ios` currently. Can we try that as an option before vendording?